src/backend/distributed/utils/metadata_cache.c

```diff
@@ -98,6 +101,10 @@ static Oid distTransactionRelationId = InvalidOid;
 static Oid distTransactionGroupIndexId = InvalidOid;
 static Oid extraDataContainerFuncId = InvalidOid;
 
+/* Citus installed extension version */
+static char *citusAvailableVersion = NULL;
+static char *citusCatalogVersion = NULL;
```

Comment:
`installedExtensionVersion` is what this should be, and should store a full string like `6.1-10`.